content/en/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md

```diff
@@ -163,3 +163,7 @@ sudo apt-get purge docker-ce docker-ce-cli
 {{% /tab %}}
 {{< /tabs >}}
 
+The above commands don't remove images, containers, volumes, or customized configuration files on your host.
+To delete them, see the [Uninstall Docker Engine](https://docs.docker.com/engine/install/ubuntu/#uninstall-docker-engine) section
+on docker docs.
+Note that it has a risk of deleting containerd, be careful when executing commands on that docs page.
```

Comment:
```suggestion
{{< note >}}
It has a risk of deleting containerd, be careful when executing commands on that docs page.
{{< /note >}}
```

Reply:
I'd actually use a `{{< caution >}}` here as the consequences for doing the wrong thing on every node could be rather unfortunate.